.github/workflows/collector-image.yml

```diff
@@ -1,13 +1,14 @@
 name: Collector image build
 
 on:
-  push:
-    branches:
-      - master
-      - release-*
-  pull_request:
-    branches:
-      - master
+  #  push:
+  #    branches:
+  #      - master
+  #      - release-*
+  #  pull_request:
+  #    branches:
+  #      - master
```

Comment:
I think we can safely remove these, we probably only want the `main.yml` to be called on pushes and PRs.